.gitignore

```diff
@@ -105,6 +105,7 @@ scripts/tenderly.ts
 # foundry
 out
 cache_hardhat
+broadcast
```

Comment:
this is where forge script command puts info about the txns it's creating and broadcasting, so want to gitignore